rerun_py/rerun_sdk/rerun/_rerun2/components/_overrides/depth_meter.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+import pyarrow as pa
+
+if TYPE_CHECKING:
+    from .. import DepthMeterArrayLike
+
+
+def depthmeter_native_to_pa_array(data: DepthMeterArrayLike, data_type: pa.DataType) -> pa.Array:
```

Comment:
Tensors are mono-archetypes, and DepthMeter should be a mono-component _I think_?

Reply:
Yes. But we don't codegen the APIs differently for mono in python.